### DISCUSSION TITLE
Automated billing / pay as you go

### DISCUSSION THREAD

**Discussion by cdrxrx** [Stability-AI/stability-sdk#59] [General]
I'd rather enter my company credit card details once and for all and "pay as I go" (even if that means automatically buy another X credits every time it's needed).
Is it supported at this stage?
Thanks a lot for your help.

EDIT: there's no pay-as-you-go for now, but a "post-paid" option is unlockable if needed.

**Comment by SeeBeNowTech** [comment]
Yes I agree much easier not sure why this was the chosen path but assuming as SDAI grows it will be easier to integrate.
